dist/az-icons-styles.css

```diff
@@ -85,3 +85,6 @@
 .az-icon-vimeo:before {
   content: "\e913";
 }
+.az-icon-x:before {
+  content: "\e914";
+}
```

Comment:
```suggestion
}

```

Reply:
Looks like there still isn't any new line at the end of the file.